packages/integration-karma/test/component/spellcheck-attribute/index.spec.js

```diff
@@ -0,0 +1,24 @@
+import { createElement } from 'lwc';
+
+import Container from 'x/container';
+
+function getSpellcheckValue(container, selector) {
+    const elm = container.shadowRoot.querySelector(selector);
+
+    return elm.getAttribute('spellcheck');
+}
+
+describe('setting static values on custom elements', () => {
+    it('should render the correct attribute value', () => {
+        const elm = createElement('x-container', { is: Container });
+        document.body.appendChild(elm);
+
+        expect(getSpellcheckValue(elm, '.empty-v')).toBe('true');
```

Comment:
This doesn't look right to me. We currently don't render global attributes as content attributes when they're set to an empty string because we set them internally as a property. We should be consistent and not render the `spellcheck` content attribute when it's set to an empty string.

Reply:
mm, i thought that we only differentiate with `undefined` or `null` and in the case of empty-string, we set the property to `''` and let the element to do the proper thing.

@ekashida If we do as you say (and i believe you are right), then we have a new issue:

```
<div hidden=""> <!-- div is hidden -->
  <textarea></textarea>
</div>
```

```
<c-custom-element hidden=""> <!-- custom-element is not hidden -->
</c-custom-element>
```

that aside, the problem with spellcheck is that if we don't render the attribute, it might not be set as true, when it should. ex:

```
<div spellcheck="false">
   <x-simple spellcheck=""></x-simple> <!-- if the attribute is not set, then elm.spellcheck === false -->
</div>
```